app/(auth)/sign-in/page.tsx

```diff
@@ -1,8 +1,137 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import Link from "next/link";
+import { useSignInMutation } from "@/redux/apis/auth-api";
+import { toast } from "sonner";
+import SocialSignUp from "@/components/social-sign-up";
+import { useAppDispatch } from "@/redux/hooks";
+import { loginUser } from "@/redux/slice/user-slice";
+import { useRouter } from "next/navigation";
+import { ROUTES } from "@/utils/constants";
 
 const SignIn: NextPage = () => {
-  return <div>SignIn</div>;
+  const router = useRouter();
+  const dispatch = useAppDispatch();
+
+  const {
+    fields: { email, password },
+  } = signInForm;
+
+  const [signIn, { isLoading }] = useSignInMutation();
+
+  const form = useForm({
+    resolver: yupResolver(signInValidationSchema),
+    defaultValues: signInDefaultValues,
+  });
+
+  const onSubmit = async (formValues: SignInFormDetails) => {
+    try {
+      const res = await signIn(formValues).unwrap();
+      toast.success("Login successfull");
+      if (res?.status === 200) {
+        dispatch(loginUser(res?.data));
+        if (res?.data?.isOrg) {
+          setTimeout(() => router.replace(ROUTES.DASHBOARD), 500);
+        } else {
+          setTimeout(() => router.replace(ROUTES.REGISTRATION_TYPE), 500);
+        }
+      }
+    } catch (err: any) {
+      toast.error(err?.data?.message);
+    }
+  };
+
+  return (
+    <AuthenticationPageTemplate redirectTo="sign-up">
+      <Form {...form}>
+        <form
+          onSubmit={form.handleSubmit(onSubmit)}
+          className="flex flex-col gap-6"
+        >
+          <FormField
+            control={form.control}
+            name={"email"}
+            render={({ field }) => (
+              <FormItem>
+                <Label htmlFor={email.name} required={true}>
+                  {email.label}
+                </Label>
+
+                <FormControl>
+                  <Input
+                    placeholder={email?.placeholder}
+                    type={email?.type}
+                    {...field}
+                  />
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+
+          <div>
+            <FormField
+              control={form.control}
+              name={"password"}
+              render={({ field }) => (
+                <FormItem>
+                  <Label htmlFor={password?.name} required={true}>
+                    {password?.label}
+                  </Label>
+
+                  <FormControl>
+                    <Input
+                      placeholder={password.placeholder}
+                      type={password.type}
+                      isPassword={true}
+                      {...field}
+                    />
+                  </FormControl>
+                  <FormMessage />
+                </FormItem>
+              )}
+            />
+
+            <Link href={"/"} className="mt-1.5 inline-block text-sm text-gray">
```

Comment:
@imrankhan-stixor , i am getting error by using `#`, i don't want to waste time in investigating this.

Reply:
This will redirect the user to some other page which isn't optimal in current case i think, I just implemented the same which jira and using "#" works